### ISSUE TITLE
ASP NET sample in Youtube SDK.msi does not work, "The remote server returned an error: (403) Forbidden."

### ISSUE THREAD

**Issue by GoogleCodeExporter** [WhatSupYap/google-gdata#337]
```
What steps will reproduce the problem?
1. Install Youtube SDK.msi
2. Rename "ASP Sample" -> "YoutubeSample" (No space in the folder name)
3. Open YoutubeSample website in VS.2008
4. Run the sample, login, grant the access, then "The remote server 
returned an error: (403) Forbidden." appear.


What is the expected output? What do you see instead?
I expect to the list of my videos, but the exception page instead.

Please use labels and text to provide additional information.
I found that the new version of Google.GData.*.dll caused the problem. 
In "ASP Sample\bin" folder, the version is 1.4.2.30022, I also tried the 
DLLs in Redist folder with version 1.5.*, still no luck.

But after I tried to use DLLs with the version 1.4.0.2, the sample works 
fine now.

I want to know if this is a bug in new version or is a new feature.

```

Original issue reported on code.google.com by `justin...@hotmail.com` on 20 Feb 2010 at 7:31

**Comment by GoogleCodeExporter** [comment]
```
It is certainly not a feature. I must have copied the wrong set of files 
somehow into
the MSI. 

Sorry for that. I guess if you actually open the website in VS, recompile 
everything
and recreate the reference everything is working fine again. I am travelling 
right
now, so i have no time to verify. Give me a week.

```

Original comment by `fman...@gmail.com` on 22 Feb 2010 at 11:13
- Changed state: **re**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fman...@gmail.com` on 22 Feb 2010 at 11:14
- Changed state: **Reviewed**

**Comment by GoogleCodeExporter** [comment]
```
The YouTube ASP.NET sample included in the latest release (1.7) is working 
correctly, please update this bug if you are still not able to run it correctly.

```

Original comment by `ccherub...@google.com` on 16 Dec 2010 at 12:25
- Changed state: **Fixed**
